training/model_management/environments/model-management/context/requirements.txt

```diff
@@ -2,6 +2,7 @@ azure-core=={{latest-pypi-version}}
 azure-ai-ml=={{latest-pypi-version}}
 azure-identity=={{latest-pypi-version}}
 azureml-evaluate-mlflow=={{latest-pypi-version}}
+langcodes=={{latest-pypi-version}}
```

Comment:
Is this needed for existing set of packages?

Reply:
required for download component